Raspberry kit support pages added (#1)
amperka/raspberry_support: docs/b-plus/README.md

# OOPS.. You've got yourself a better Raspberry Pi! It's B+!

> Good news! Fellows from Raspberry managed to update their Raspberry Pi 3B controller up to Model B+ . It means that our book — the one you've found in the box, is somewhat obsolete, because it was published before the update

Reviewer: ```suggestion
Good news! Fellows from Raspberry Pi Foundation managed to update their Raspberry Pi 3 Model B controller up to Model B+. It means that our book — the one you've found in the box, is somewhat obsolete, because it was published before the update
```

Let's get to it.

> ## Page 4. Raspberry PI

Reviewer: ```suggestion
## Page 4. Raspberry Pi
```

Obviuosly, Raspberry Pi 3B+ looks different:

> ![Image](/docs/b-plus/images/pi.png)

Reviewer: ```suggestion
![Image](./images/pi.png)
```

Use relative paths here and everywhere below
